ChangeLog.md

```diff
@@ -34,45 +79,49 @@
 + Re-added item shadows
 + Re-added swimming animation
 + Re-added colored spawners
-* Fixed crash when inputting certain characters into the inventory searcher
-* Fixed crash when deleting world
-* Fixed rendering of stair tiles in sky level
-* Fixed rendering of cloud cactus in sky level
-* Fixed time rendering for potions not following m:ss format
-* Fixed some hits not hitting properly (0 attack damage)
+
+#### Changes
+
 * Made gem tools purple
 * Prevented animals and mobs from de-spawning while in view
 * Prevented furniture from being pushed over stair tiles
 * Shear loses durability when used on sheep
-* The world list should now update if copying or renaming worlds in-game
 * Resource packs and skins menu now use the standardised menu look
 * Pre-releases are now ignored from version check
-* Renamed shear to shears
 * Saplings should no longer grow if an entity is on it
+* Renamed shear to shears
 * Made cloud cactus into an ore you can mine and use
+
+#### Removals
+
 - Removed texture packs
 - Removed suits
 - Removed second air wizard
 
-## 2.1.1
-* Fixed air wizard spawner
-* Fixed player spawning outside the map
+#### Fixes
 
-## 2.1.3
-* Fixed localization not being loaded on first launch
-- Removed option of changing skin in the options display because of a crash
+* Fixed crash when inputting certain characters into the inventory searcher
+* Fixed crash when deleting world
+* Fixed rendering of stair tiles in sky level
+* Fixed rendering of cloud cactus in sky level
+* Fixed time rendering for potions not following m:ss format
+* Fixed some hits not hitting properly (0 attack damage)
+* The world list should now update if copying or renaming worlds in-game
+
+> ---------------------------------------------------------------------------------------
+>
+> Changed versioning system to be in compliance with https://semver.org/spec/v2.0.0.html
+>
+> ---------------------------------------------------------------------------------------
```

Comment:
Don't change anything below this

Reply:
I don't understand.

Reply:
Just don't change anything below this line.

Reply:
Why? They don't suit the format.

Reply:
https://github.com/MinicraftPlus/minicraft-plus-revived/pull/412#discussion_r1032923955